README.md

```diff
@@ -122,7 +121,7 @@ __Properties__
 
 ### Caesar
 ![alt text][caesar]<br>
-In cryptography, a **Caesar cipher**, also known as Caesar's cipher, the shift cipher, Caesar's code or Caesar shift, is one of the simplest and most widely known encryption techniques.<br>
+In cryptography, a **Caesar cipher**, also known as *Caesar's cipher*, the *shift cipher*, *Caesar's code* or *Caesar shift*, is one of the simplest and most widely known encryption techniques.<br>
```

Comment:
I think this can be removed 'In cryptography, a '. It can start directly from Caesar Cipher, also known as ...